core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -2673,7 +2674,7 @@ public void testPreparedStatementWithBindVariablesSetWrongOnDifferentConnection(
                 }
 
                 try (final Connection connection = getConnection(false, false);
-                     PreparedStatement statement = connection.prepareStatement("drop table xts")) {
+                     PreparedStatement statement = connection.prepareStatement("drop table if exists xts")) {
```

Comment:
We must not change existing tests to conform with new implementation. Existing tests to ensure non-regressive behaviour.

Reply:
I have added a test in DropTableTest as well